main.py

```diff
@@ -159,30 +164,28 @@ def song_format(
     Returns:
         A string presenting the given song information in a human-readable way.
     """
-    # a valid tag is string with at least one non-whitespace character
-    def is_valid_tag(tag: Optional[str]) -> bool:
-        return tag is not None and tag.strip()
-
     content = ""
-    tags = None
 
     # load tags
     try:
-        tags = TinyTag.get(local_filepath)
-    except TinyTagException:
-        # Ignore and move on
-        pass
+        tags = MutagenFile(local_filepath)
+        if isinstance(tags, ID3FileType):
+            # ID3 keys are complicated. EasyID3 unifies the interface with other types
+            tags = EasyID3(local_filepath)
```

Comment:
You can actually provide `easy=True` to `MutagenFile` in order to do this automatically. This also has the advantage of returning the `EasyMP3`, `EasyMP4` and `EasyTrueAudio` interfaces as well. [Relevant mutagen source](https://github.com/quodlibet/mutagen/blob/30f373fa6d56e1afa17d48a0c6f3e111267fbd32/mutagen/_file.py#L238-L259).